src/nanoemoji/config.py

```diff
@@ -113,6 +113,9 @@ class FontConfig(NamedTuple):
     color_format: str = "glyf_colr_1"
     upem: int = 1024
     width: int = 1024
+    # TODO: How OpenType Works says:
+    # The sTypoAscender minus the sTypoDescender should equal the unit square.
+    # Do we need to adjust the default value here?
     ascent: int = 950  # default based on Noto Emoji
     descent: int = -250  # default based on Noto Emoji
```

Comment:
@rsheeter PTAL

Reply:
@anthrotype thoughts? - I'll note Noto does not appear to follow this rule

Reply:
> sTypoAscender minus the sTypoDescender should equal the unit square

not true, that recommendation is bogus. One can set them to whatever makes sense for the design at hand.

Reply:
It does not need to be equal the unit square.

I found some introductions about [UPM dogma](https://docs.microsoft.com/en-us/typography/opentype/spec/os2#stypoascender). And the description of the [opentype spec](https://docs.microsoft.com/en-us/typography/opentype/spec/os2#stypoascender): 
> t is not a general requirement that sTypoAscender - sTypoDescender be equal to unitsPerEm. These values should be set to provide default line spacing appropriate for the primary languages the font is designed to support.

Reply:
Comment removed.